spec.txt

```diff
@@ -3485,7 +3476,6 @@ can occur one after another, without intervening blank lines.
 [bar]: /bar-url
   "bar"
 [baz]: /baz-url
-
 [foo],
```

Comment:
Is this change needed?

Reply:
It's not strictly needed, but it's in a section about being able to omit intervening blank lines, so I dropped the blank line to emphasize that in another context.